src/System.Net.Security/src/System/Net/NegotiateStreamPal.Unix.cs

```diff
@@ -19,11 +22,20 @@ namespace System.Net.Security
     //
     // This is part of the NegotiateStream PAL.
     //
-    internal static class NegotiateStreamPal
+    internal partial class NegotiateStreamPal
     {
+        private const int LogonDenied = unchecked((int)0x8009030C);
+        private const int NTE_FAIL = unchecked((int)0x80090020);
```

Comment:
LogonDenied  is anyways gone by moving code to common . added comment for another/